sdk/java-sdk-spring/src/test/scala/kalix/javasdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -533,7 +533,7 @@ class ActionDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSu
         eventingIn.getIgnore shouldBe false
         eventingIn.getIgnoreUnknown shouldBe false
 
-        val methodDescriptor = findMethodByName(desc, "KalixSyntheticMethodOnESEmployee_events")
```

Comment:
Well, I don't think there is any backward-compatibility issue here. This is just the generation of the method name in protobuf. This is transparent to the users.

Reply:
Isn't this used as a part of the projection name on the proxy side? I have a feeling that this might reset the projection offset, which is not a big deal because no one is using this SDK on production anyway.

Reply:
I don't think that's correct. This defines the name of the method that will receive the payload. 

Imagine an action subscribing to an event stream with events like `Created`, `ItemAdded` and `CheckedOut`. You will need three methods to receive each event, but the offset should be tracked by a unique id and that should be independent of the methods receiving it.

True that in the case of the Java (code-first) SDK we have only one method and the branching is done on the SDK side, but the proxy is unaware of it. From the proxy perspective, a new method can be added at any time. For example, when a new event type is introduced. And that should never be an issue or affect the offset tracking in any away.